### PR TITLE
adding spring prop in readme

### LINKED ISSUE #34: adding Environment variables / spring properties for each service

### PR DESCRIPTION
**Pull request by Keyshal-211**
## Description of changes
Please provide a short description of the change or if the change is sufficiently described in the GitHub issue, provide a link to the issue.

  
## How has this been tested?
Please describe the test strategy you followed.

- [ ] automated unit test
- [ ] automated integration test
- [ ] automated acceptance test
- [ ] manual, exploratory test

In case of manual test, please document the test well including a set of user instructions and prerequisites. Each including an action, it's result, and where appropriate a screenshot.
In case of automated tests, the test coverage should be reasonably high, especially on Service classes and other classes with complex logic.
    
## Checklist before requesting a review
- [X] My code follows the [coding guidelines](https://github.com/IT-REX-Platform/wiki/blob/main/dev-manuals/backend/coding-guidelines.md) of this project
- [X] I have performed a self-review of my code
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] My code fulfills all acceptance criteria
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] The test coverage (line and branch) is reasonably high, especially on Service classes and other classes with complex logic
- [ ] I have made corresponding changes to the documentation
- [ ] I have added explanation of architectural decision and rationales to [wiki/adr](https://github.com/IT-REX-Platform/wiki/tree/main/adr)
- [ ] I have updated the changes in the ticket description

## Checklist for reviewer
- [ ] The code works and does not throw errors
- [ ] The code is easy to understand and there are no confusing parts
- [ ] The code follows the [coding guidelines](https://github.com/IT-REX-Platform/wiki/blob/main/dev-manuals/backend/coding-guidelines.md) of this project
- [ ] The code change accomplishes what it is supposed to do
- [ ] I cannot think of any use case in which the code does not behave as intended
- [ ] The added and existing tests reasonably cover the code change
- [ ] I cannot think of any test cases, input or edge cases that should be tested in addition
- [ ] Description of the change is included in the documentation

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,6 +6,34 @@ This service is responsible for calculating and storing the skill levels student
...
+| spring.sql.init.continue-on-error         | Continue on SQL init error                | true                                                           | true                                    |
+| spring.jpa.hibernate.ddl-auto             | Hibernate DDL auto strategy               | create                                                         | update                                  |
+| hibernate.create_empty_composites.enabled | Enable empty composite types in Hibernate | true                                                           | true                                    |
+| DAPR_GRPC_PORT                            | Dapr gRPC Port                            | -                                                              | 50001                                   |
+| logging.level.root                        | Logging level for root logger             | INFO                                                           | -                                       |
+| content_service.url                       | URL for content service GraphQL           | http://localhost:3500/v1.0/invoke/content-service/method/graphql | http://app-content:4001/graphql         |
```
Comment (PaulBredl) [inline README.md]: This is relevant for deployment

**Review by PaulBredl — APPROVED** [review]
